src/server/FastAPI_server.py

```diff
@@ -0,0 +1,104 @@
+"""
+走行体と通信するWebサーバー.
+
+@author Hara1274
+"""
```

Comment:
細かいけど、ファイル名は`fastapi_server.py`じゃだめ？
```suggestion
"""
走行体と通信するWebサーバープログラム

@file fastapi_server.py
@author Hara1274
"""
```